integration-test/src/test/java/com/datastrato/gravitino/integration/test/spark/SparkIT.java

```diff
@@ -315,6 +315,32 @@ void testAlterTableUpdateColumnType() {
     checkTableColumns(tableName, updateColumns, getTableInfo(tableName));
   }
 
+  @Test
+  void testUpdateColumnPosition() {
+    String tableName = "test_column_position";
+    dropTableIfExists(tableName);
+
+    List<SparkColumnInfo> simpleTableColumns = getSimpleTableColumn();
+
+    createSimpleTable(tableName);
+    checkTableColumns(tableName, simpleTableColumns, getTableInfo(tableName));
+
+    sql(String.format("ALTER TABLE %S ADD COLUMNS (col1 string)", tableName));
+    sql(String.format("ALTER TABLE %S CHANGE COLUMN col1 col1 string FIRST", tableName));
```

Comment:
I prefer to test `last` case because this is most used.

Reply:
I will perfect the test cases for first, after and default position.

Reply:
updated.